### ISSUE TITLE
Add learning rate scheduler (+warm up scheduler)

### ISSUE THREAD

**Issue by Elliot-D-Hill** [duke-aihealth-ds-fellowship/pytorch-template#7]
Title
